package.json

```diff
@@ -1,13 +1,12 @@
 {
-  "name": "@vaadin/react-components",
-  "version": "24.4.0-alpha8",
-  "description": "React wrappers for Vaadin components",
+  "name": "react-components-workspace",
   "type": "module",
-  "module": "index.js",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "version": "24.4.0-alpha8",
+  "private": true,
   "scripts": {
```

Comment:
The build is pretty confusing:
- Most NPM scripts don't work without setting an env variable
- Most NPM scripts are generic, but some are package specific, naming is also challenging
- Scripts in `scripts` use an env variable to determine which folder to operate in, which is kind of hidden in a utility module

Would it be feasible to duplicate the NPM scripts for the build into the respective packages and then just run the build separately inside each package folder? The modules within `script` could then just use the current folder as context instead of an env variable. Duplicating the NPM scripts between two packages might be a better trade-off compared to the points above.

Reply:
Good idea, this approach makes more sense than using the environment variables. Done.